src/app/components/Signup.tsx

```diff
@@ -24,23 +23,9 @@ const Signup = () => {
     }
     try {
       setError(null);
-      setAlert();
+      setAlert(null);
       setLoading(true);
-      await signup(emailRef.current.value, passwordRef.current.value).then(
-        userCredential => {
-          setDoc(doc(usersRef, userCredential.user.uid), {
-            first: '',
-            last: '',
-            street: '',
-            houseapt: '',
-            city: '',
-            postalcode: '',
-            country: '',
-            phone: '',
-            email: emailRef.current.value,
-          });
-        },
-      );
+      await signup(emailRef.current.value, passwordRef.current.value);
       setAlert(`You signed up! :) Wait to be redirected to Log In page`);
```

Comment:
Wondering about extract all the errors to const(above the component) or utils. What do you think?